retro-lcd/usr/share/asteroid-launcher/watchfaces/retro-lcd.qml

```diff
@@ -81,10 +151,18 @@ Item {
             ctx.textAlign = "center";
             ctx.font = "50 " + fontSizeCalc(46) + "px " + "Roboto";
             ctx.fillText(":", centerX - (centerX * 0.103), centerY + (centerY * 0.15));
+ 
         }
     }
     Connections {
         target: wallClock
-        onTimeChanged: digital.requestPaint()
+        onTimeChanged:{
+	digital.requestPaint()
```

Comment:
Could you fix the indentation here? :) (maybe your code editor show them the same but it's typical that tabs/spaces are render different on other apps, like github here)